src/Paket.Core/TemplateFile.fs

```diff
@@ -6,6 +6,80 @@ open System.IO
 open System.Text.RegularExpressions
 open Paket.Rop
 open Paket.Domain
+
+module private TemplateParser =
+    type private ParserState =
+        {
+            Remaining : string list
+            Map : Map<string, string>
+            Line : int
+        }
+
+    let private single = Regex("^(\S+)\s*$", RegexOptions.Compiled)
```

Comment:
Out of curiosity, why isn't the regex pattern a verbatim string? Shouldn't the backslashes be escaped? I'm no F# expert, so I could be totally wrong here.